starter-code/src/foodbox.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
+import './App.css';
+import food from './foods.json'
+
+
+class FoodBox extends Component {
+    constructor(props){
+        super(props)
+
+        this.state = {
+          countItems : 1
+        }
+    }
+
+    AddCountItems = () => {
+      this.setState({
+        countItems : this.state.countItems + 1
```

Comment:
You should retrieve the value in the event object. You are always adding +1 to the state in charge of your input, which is why it always goes up one value once you click. Also, just so you know: 
```js
// When a setState depends on previous state, you can use a function parameter:
this.state = { counter: 1 }

this.setState((prevState, props) => {
	return {
		counter: prevState.counter + 1
	}
})
```